README.md

```diff
@@ -100,6 +100,8 @@ Head over to [asyncapi/raml-dt-schema-parser](https://www.github.com/asyncapi/ra
 
 The parser API is generally structured the same way as the AsyncAPI specification, with additional support functions such as `has<Something>()`. The parser uses wrapped functions to get the properties stored in JSON. This means in order to get the info object you would use the function `doc.info()` and to get the title inside the info object you call `doc.info().title()`.
 
+The parser by default supports on AsyncAPI Schema Format and JSON Schema Format. For additional formats, you need to plugin additional plugins.
```

Comment:
Please move it to the `Install` section under installation instruction and provide a code block under this test, that also explains how to install plugins:

```js
npm install @asyncapi/avro-schema-parser
//and the rest
```

Reply:
Hi @derberg, I updated the README.md file. Please check.